lib/bdd/rspec/example_group.rb

```diff
@@ -1,77 +1,70 @@
 module Bdd
   module RSpec
-    module ExampleGroup
+    module ExampleGroupSteps
+      module ClassMethods
+        def define_step(*names)
+          steps.concat(names)
+          names.each do |name|
+            define_method(name) do |msg, &block|
+              step([name, msg], &block)
+            end
+          end
+        end
+        def steps
+          @steps
+        end
+      end
+
+      def self.included(base)
+        # initialize empty steps
+        base.instance_variable_set(:@steps, [])
+        # step needs access to the @steps - but it is not visible when extend is used, especially not as @@steps
+        base.send(:define_method, :steps) { base.instance_variable_get(:@steps) }
+        # add the `define_step` and `steps` class methods
+        base.extend ClassMethods
+      end
 
-      def step(array, &block)
+      def step(prefix_and_text, &block)
```

Comment:
could you please also rename this method to `bdd_step` so we don't run into any conflicts with other gems?